src/js/components/utils/editModal.tsx

```diff
@@ -0,0 +1,122 @@
+import Button from '@salesforce/design-system-react/components/button';
+import Input from '@salesforce/design-system-react/components/input';
+import Modal from '@salesforce/design-system-react/components/modal';
+import Textarea from '@salesforce/design-system-react/components/textarea';
+import i18n from 'i18next';
+import React, { useCallback, useEffect, useRef } from 'react';
+
+import { useForm } from '@/components/utils';
+import { Project } from '@/store/projects/reducer';
+
+interface EditModalProps {
+  project: Project;
+  isOpen: boolean;
+  toggleModal: (open: boolean) => void;
+}
+const EditModal = ({ project, isOpen, toggleModal }: EditModalProps) => {
+  const form = useRef<HTMLFormElement | null>(null);
```

Comment:
Why do we need the `form` as a ref?

Reply:
@jgerigmeyer using a ref since we are displaying the form with the current name and description using a [default value](https://reactjs.org/docs/uncontrolled-components.html#default-values) from the store.  Is there a way to do this with controlled components?

Reply:
Yes, that's the intention of `useForm` (which tracks the values via component State), and that's how we're using it for other forms throughout. We definitely want these to be controlled components. Instead of `defaultValues`, we can pass initial values through `fields` arg to useForm.

Reply:
See `submitReview.tsx` as an example of passing initial values through to `useForm`.